ansible/config_sonic_basedon_testbed.yml

```diff
@@ -35,6 +35,81 @@
   tasks:
 
   - block:
+    - name: Creates telemetry directory
+      file:
+          path: /etc/sonic/telemetry
+          state: directory
+      become: true
+
+    - name: Init telemetry keys
+      set_fact:
+          server_key: ""
+          server_csr: ""
+          dsmsroot_key: ""
+          dsmsroot_csr: ""
+
+    - name: read server key
+      set_fact:
+          server_key: "{{ telemetry_certs['server_key'] }}"
+      when: telemetry_certs['server_key'] is defined
+
+    - name: read server csr
+      set_fact:
+          server_csr: "{{ telemetry_certs['server_csr'] }}"
+      when: telemetry_certs['server_csr'] is defined
+
+    - name: read dsmsroot key
+      set_fact:
+          dsmsroot_key: "{{ telemetry_certs['dsmsroot_key'] }}"
+      when: telemetry_certs['dsmsroot_key'] is defined
+
+    - name: read dsmsroot csr
+      set_fact:
+          dsmsroot_csr: "{{ telemetry_certs['dsmsroot_csr'] }}"
+      when: telemetry_certs['dsmsroot_csr'] is defined
+
+    - name: Create telemetry server private key
+      openssl_privatekey:
+          path: "{{ server_key }}"
+          size: 2048
+      become: true 
```

Comment:
should the telemtry server private key on the dut? should this key go to the ptf docker, where the server will be?

Reply:
The telemetry server private key is needed on the server side. On ptf docker client side cert will be needed which in this case will be "dsmsroot.cer" basically plan is to use this cert on ptfdocker side . Any better suggestions?

Reply:
which on is the server, the sonic or the ptf docker?

Reply:
sonic is the server and ptfdocker will have gnmi_get(client side)

Reply:
How will this dsmsroot.cer and ky be copied onto ptf docker? And you will need renaming as well. Will it be handled in this PR or another PR?

Reply:
added support to copy on ptf host

Reply:
added support to copy client certs on ptf docker